<?xml version="1.0"?>
<launch>
   <arg name="mode" default="real"/> <!--sim/real-->
   <arg name="exp_type" default="regular"/>
   <arg name="config_time" default="20" />
   <arg name="fixed_height" default="-0.04" />

  <node name="exp_ltm_17" pkg="mdb_baxter_experiments" type="exp_ltm_17.py" required="true" output="screen">
    <param name="mode" type="str" value="$(arg mode)" />
	<param name="exp_type" type="str" value="$(arg exp_type)" />
	<param name="config_time" type="int" value="$(arg config_time)" />
	<param name="fixed_height" type="double" value="$(arg fixed_height)" />
  </node>

  <node name="exp_ltm_helper" pkg="mdb_baxter_experiments" type="exp_ltm_helper.py" required="true" output="screen"/> 
</launch>
